uwufetch.c

```diff
@@ -441,7 +441,6 @@ void uwu_pkgman(char* pkgman_name) {
   PKGMAN_TO_UWU("flatpak", "fwatpakkies");
   PKGMAN_TO_UWU("pacman", "pacnyan");
   PKGMAN_TO_UWU("port", "powt");
-  PKGMAN_TO_UWU("rpm", "rawrpm");
   PKGMAN_TO_UWU("snap", "snyap");
   PKGMAN_TO_UWU("zypper", "zyppew");
```

Comment:
Shouldn't you remove this line instead of 444?

Reply:
i forgot to remove 446, but removing 444 also makes sense, because now there is no "not raw rpm" (zypper)

Reply:
oh, sorry, i meant to edit 444